dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/secret/ServiceIntegrationResourceTest.java

```diff
@@ -134,6 +138,9 @@ public void Test_Create_service_descriptor_Then_Create_Service_Integration_Then_
         );
         final HttpServletRequest request = mock(HttpServletRequest.class);
         final HttpServletResponse response = mock(HttpServletResponse.class);
+
+        when(request.getRequestURI()).thenReturn("/baseURL");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "/baseURL" appears 7 times in this file; the first occurrence is on line 142](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4960082)